docs/explanation/EXECUTE_COMMAND_CONFIGURATION_GUIDE.md

```
@@ -170,16 +170,16 @@ EOF
 cargo test -p perl-parser --test execute_command_tests -- test_run_builtin_critic_arithmetic_mutations
 
 # Validate <300ms performance target for files <100KB
-cargo test -p perl-lsp --test lsp_performance_tests -- test_execute_command_latency
+cargo test -p perl-lsp-rs --test lsp_performance_tests -- test_execute_command_latency
 ```
 
 4. **Configure LSP timeouts**:
```

markdownlint-cli2 (0.22.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 4; Style: 1/1/1

(MD029, ol-prefix)